TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift

```diff
@@ -162,6 +156,23 @@ class KeychainQueryTests: XCTestCase {
     XCTAssertEqual(data, expectedData)
     XCTAssertEqual(accessGroup, Constants.accessGroup)
   }
+
+  func testSaveItem_withAttrAccessible_shouldReturnValidQuery() {
+    keychainQuery = KeychainQuery(accessGroup: Constants.accessGroup)
+    let expectedData = "data".data(using: .utf8)!
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/1230259fcee173b2bad2b801de0a522beecabdd9/TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift#L162"></span>Prefer non-optional <code>Data(_:)</code> initializer when converting <code>String</code> to <code>Data</code><br />
<code>non_optional_string_data_conversion</code> <code>KeychainQueryTests.swift:162</code></td>
    </tr>
  </tbody>
</table>